client/src/components/Chat.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { ChatBubble } from './ChatBubble';
+
+export const Chat = () => {
+  const messages = [
+    {
+      type: 'rcvd',
+      text: 'Lorem ipsum',
+      timestamp: '11:52',
```

Comment:
Data should be modeled as what it is, whenever possible. Instead of a string, something like luxon-date's DateTime (a few other options). Using strings for everything causes ambiguity and more potential for things to arrive in a bad state